README.rst

```diff
@@ -197,482 +197,773 @@ automatically fix anything:
 Choose from the list of available rules:
 
 * **array_syntax**
-   | PHP arrays should be declared using the configured syntax (requires PHP
-   | >= 5.4 for short syntax).
-   | *Rule is: configurable.*
+
+  PHP arrays should be declared using the configured syntax (requires PHP
+  >= 5.4 for short syntax).
+
+  Configuration options:
+
+  - ``syntax`` (``'long'``, ``'short'``): whether to use the ``long`` or ``short`` array
+    syntax; defaults to ``'long'``
 
 * **binary_operator_spaces** [@Symfony]
-   | Binary operators should be surrounded by at least one space.
-   | *Rule is: configurable.*
+
+  Binary operators should be surrounded by at least one space.
+
+  Configuration options:
+
+  - ``align_double_arrow`` (``NULL``, ``false``, ``true``): whether to apply, remove or
+    ignore double arrows alignment; defaults to ``false``
+  - ``align_equals`` (``NULL``, ``false``, ``true``): whether to apply, remove or ignore
```

Comment:
It should be `null, false, true`, not `NULL, false, true`.

PS yes, `var_export` is stupid